src/transform/transform-strict-mode.js

```diff
@@ -0,0 +1,25 @@
+import { types as t } from '@babel/core';
+
+const visitor = {
+  Program: programVisitor,
+};
+
+function programVisitor(path, state) {
+  const {
+    node: { directives },
+  } = path;
+
+  for (const directive of directives) {
+    if (directive.value.value === 'use strict') return;
+  }
```

Comment:
저 이쪽에 궁금한 점이 있는데요
여기 for loop 는 directives 내에 'use strict' value 를 가지는 게 있는지만 판단하는 용도인가요? 
if 문에서 해당하는 값을 찾았을 때 return 만 하구 있어서요 !

Reply:
네 맞습니다.
for문내부에서 원하는 value값을 찾으면 for문뿐만 아니라 함수자체를 종료시키기 위해서 return문을 추가했어요.

Reply:
중요한건 아니지만 for loop 에서 리턴 시키는 것 보다 배열 메서드 써서 필터링하는게 좀 더 깔끔할 것 같은데 이랑 맘에 드는 거로 하세여 ㅎ

```suggestion
  const isExistUseStrictValue = directives.some(
    (directive) => directive.value.value === 'use strict'
  );

  if (!isExistUseStrictValue) {
    path.unshiftContainer(
        'directives',
        t.directive(t.directiveLiteral('use strict')),
    );
  }
```

Reply:
무엇을 하려는지 의도가 드러나서 더 좋은 방법인 것 같아요..!
꿀팁 감사합니다 : )